### PR TITLE
Added python opensearch-logger community project

### PR DESCRIPTION
**Pull request by vduseev**
### Description
Adding the Opensearch logging handler for python as a community project.
 
### Issues Resolved
None

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _community_projects/python-opensearch-logger.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Python Logger
+description: Opensearch logger for Python sends standard logs directly to Opensearch. It provides direct and easy to use logging handler. Supports out of the box Python logging.
```
Comment (elfisher) [inline _community_projects/python-opensearch-logger.md:3]: minor request to use "OpenSearch" casing.

**Review by elfisher — CHANGES_REQUESTED** [review]
This LGTM. One minor change requested. Thanks @vduseev

**Comment by vduseev** [comment]
@elfisher thanks for noticing. My bad, I thought OpenSearch is written like "Opensearch" 😄

**Review by elfisher — APPROVED** [review]
Hi @vduseev thank you for being part of the OpenSearch project and building a Python client library for handling logging! I’m excited to get this project highlighted on the community project page. With that said, I did want to make a note regarding the logo provided as it is not exactly in line with the project brand guidelines. As a best practice to help keep the OpenSearch brand unified and cohesive, the logo should be used in accordance with the guidelines. However, in this case the logo you’ve provided will be permitted, but I will ask that you use it when specifically referencing this Python client library and not the overall project as that may confuse users. Thanks!

**Comment by elfisher** [comment]
@stockholmux do we have any other website content we aim to release? I wanted to make sure I didn't merge if there was coordination needed.

**Comment by vduseev** [comment]
@elfisher thank you for accepting the project into the community page. I will make sure to only use logo in reference to this python based logger compatible OpenSearch, not the OpenSearch project as a whole. 

I can also alter the logo by adding something, like an icon in the front, if you think that's a good idea. That would indicate that this is a logger and not the `opensearch-py` library. The current logo is just the OpenSearch logo colored into the python gradient colors.

**Review by stockholmux — APPROVED** [review]
LGTM

**Comment by stockholmux** [comment]
@vduseev Content is totally fine, however it looks like you didn't do a commit sign off (`git commit ... -s`) to meet the DCO requirement ([good resource if you haven't had to do this before](https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits#answer-15667644)).

**Comment by vduseev** [comment]
@stockholmux yeah, I didn't fully understand before what sign-off means. I have squashed both commits into one and signed it off using `git --signoff` feature. Hope I did it right :)

**Review by stockholmux — APPROVED** [review]
LGTM

**Comment by stockholmux** [comment]
@vduseev Thanks! You did it perfectly.

We've merged it into staging and it will go out in our next push live.

**Pull request merged**